lib/libspl/include/umem.h

```diff
@@ -134,10 +134,11 @@ umem_free(void* ptr, size_t size __maybe_unused)
     free(ptr);
 }
 
+
 static inline void
-umem_free_aligned(void* ptr, size_t size __maybe_unused)
+umem_free_aligned(void* ptr, size_t size)
 {
-    posix_memalign_free(ptr);
+    _aligned_free(ptr);
```

Comment:
I think I invented `posix_memaligned_free()` for Windows, which just calls _aligned_free().  and it should be just "free" for the other platforms. That is to match the `posix_memalign()` call.